docs/contribute.md

```diff
@@ -0,0 +1,50 @@
+## Contributor's Guide
+
+#### Software Requirements
+In order to contribute on the nsm operator project make sure you have the minimal software requirements below:
+
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [golang version 1.13+](https://golang.org/doc/install)
+- [operator-sdk v0.15.1](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md)
+- [podman version 1.2.0+](https://podman.io/getting-started/installation.html)
+ or [docker version 19+](https://docs.docker.com/install/)
+- GNU make (https://www.gnu.org/software/make/)
+- access to a Kubernetes 1.21+ cluster (for development it can be [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) or [kind](https://kind.sigs.k8s.io/docs/user/quick-start/))
+- access to an [OpenShift](https://docs.openshift.com/container-platform/4.3/installing/installing_aws/installing-aws-default.html) Cluster (Not tested on [CRC](https://github.com/code-ready/crc))
```

Comment:
Is openshift a requirement or?

Reply:
It should be able to install in vanilla Kubernetes without any issues. It doesn't make any use of OpenShift's api custom resources. We'll put some CI in place to test both in parallel.